vars/helm.groovy

```diff
@@ -78,10 +80,15 @@ def deployRemoteChart(credentialsId, environment, namespace, chartName, chartVer
       file(credentialsId: "$chartName-$environment-values", variable: 'values')
     ]) {
       def extraCommands = getExtraCommands(chartVersion)
-      sh "helm repo add ffc $HELM_CHART_REPO"
-      sh "helm repo update"
+      addHelmRepo('ffc', $HELM_CHART_REPO)
       sh "kubectl get namespaces $namespace || kubectl create namespace $namespace"
       sh "helm upgrade --namespace=$namespace $chartName -f $values --set namespace=$namespace ffc/$chartName $extraCommands"
```

Comment:
Just related to the comment 2 lines above, the `ffc` repo name is hardcoded on this line, just in case the name is changed above.